src/types/runtime/gravity/Feature.ts

```diff
@@ -29,6 +29,7 @@ export const Feature = Record({
     source: String.Or(Undefined),
   }).Or(Null),
   created_at: String,
+  meta_title: String.Or(Null).Or(Undefined),
```

Comment:
Funny but this is the only Gravity response we have that's properly typed and I must say: it's nice!